ledger/controllers/wallet.js

```diff
@@ -163,7 +163,9 @@ async function sumActiveGrants (runtime, info, wallet, grants) {
     } else {
       let content = braveUtils.extractJws(token)
       total = total.plus(content.probi)
-      results.push(underscore.pick(content, ['altcurrency', 'expiryTime', 'probi']))
+      const exposedContent = underscore.pick(content, ['altcurrency', 'expiryTime', 'probi', 'type'])
+      exposedContent.type = exposedContent.type || 'ugp'
```

Comment:
had an awful time with this bit. for some reason i couldn't get the type to flow through even with the joi validation applied.

Reply:
ah I assume you tried `optional()`?

Reply:
i did not, but when i tried the setup on runkit i was getting the correct result